tools/gstf_preparation/gstf_preparation.xml

```diff
@@ -36,11 +40,13 @@ python '$__tool_directory__/gstf_preparation.py'
         <param name="fasta_inputs" type="data" format="fasta" multiple="true" label="Corresponding FASTA datasets" help="Each FASTA header line should start with a transcript id" />
         <param name="longestCDS" type="boolean" checked="false" label="Keep only the longest CDS per gene" />
         <param name="headers" type="boolean" checked="true" label="Change the header line of the FASTA sequences to the &gt;TranscriptId_species format" help="As required by TreeBest, part of the GeneSeqToFamily workflow" />
+        <param name="keys" type="text" optional="true" label="List of keywords (Chromosome names) to ignore FASTA sequences" help="Use comma (,) separated keywords. i.e. MT, Chr 1, Chr X" />
```

Comment:
Can you use a different name for this parameter? It seems the filtering is not happening correctly: in the 5th test the test5.ns.fasta output file is empty and if you look at the command script generated by Galaxy (e.g. https://travis-ci.org/TGAC/earlham-galaxytools/builds/370493726#L1696 ) it seems `keys` may be a reserved word.